Website/src/rendering/src/assets/css/layout/sections.css

```diff
@@ -10,15 +10,33 @@
   }
 
   &--dark {
-    @apply bg-black;
+    background: url(../../img/background-pattern-dark.svg);
 
     .section__content {
       .section__content__p {
+        @apply block text-white;
+      }
+
+      .section__content__title {
+        @apply text-orange normal-case;
+      }
+
+      .item-grid .grid-content .grid-item .item-details {
```

Comment:
The Section CSS file should not know about the item grids. Please move these styles elsewhere.

Reply:
this is section specific
only applies here